rasa/core/channels/channel.py

```diff
@@ -356,147 +360,3 @@ async def send_custom_json(
         self, recipient_id: Text, json_message: Dict[Text, Any], **kwargs: Any
     ) -> None:
         await self._persist_message(self._message(recipient_id, custom=json_message))
-
-
-class QueueOutputChannel(CollectingOutputChannel):
-    """Output channel that collects send messages in a list
-
-    (doesn't send them anywhere, just collects them)."""
-
-    @classmethod
-    def name(cls) -> Text:
-        return "queue"
-
-    # noinspection PyMissingConstructor
-    def __init__(self, message_queue: Optional[Queue] = None) -> None:
-        super().__init__()
-        self.messages = Queue() if not message_queue else message_queue
-
-    def latest_output(self) -> NoReturn:
-        raise NotImplementedError("A queue doesn't allow to peek at messages.")
-
-    async def _persist_message(self, message) -> None:
-        await self.messages.put(message)  # pytype: disable=bad-return-type
-
-
-class RestInput(InputChannel):
-    """A custom http input channel.
-
-    This implementation is the basis for a custom implementation of a chat
-    frontend. You can customize this to send messages to Rasa Core and
-    retrieve responses from the agent."""
-
```

Comment:
Should we make a deprecation warning'd class here that imports the other one and tells the user to change their imports? We had done that with many modules we'd moved before, so that custom connectors don't break on upgrade. 

If not, we should add the change to the migration guide

Reply:
Hum, since we're launching 2.0, I don't know if it really make sense to introduce it. And the class _was_ and _is still_ importable `from rasa.core.channels import ...`

Reply:
you make a good point!